### DISCUSSION TITLE
Unable to Use GitHub Copilot

### DISCUSSION THREAD

**Discussion by FedeAletti** [community/community#58336] [Copilot Conversations]
### Copilot

Bug

### Body

Hello everyone,

I wanted to let you know that as a GitHub Student Pack holder, I am currently facing difficulties in accessing GitHub Copilot. Despite having the Student Pack benefits, I am unable to utilize this feature.

Any assistance or insights regarding this issue would be greatly appreciated. Thanks!

**Comment by OnurIlgaz** [comment]
Same issue bro
